cypress.json

```diff
@@ -27,5 +27,8 @@
   "viewportHeight": 768,
   "chromeWebSecurity": false,
   "nodeVersion": "system",
-  "retries": 1
+  "retries": {
+    "runMode": 3,
```

Comment:
yeah I don't want to increase the regular runMode above 1 because we're most likely just increasing the wait time of failing tests vs preventing flake.

Reply:
Right... we discussed that. Sorry. Will keep investigating.

Reply:
This can be merged if this is `runMode: 1` and we could keep the other card open